Games/core/VegKickMeeting.js

```diff
@@ -96,6 +104,7 @@ module.exports = class VegKickMeeting extends Meeting {
         let vegKickThreshold = Math.ceil(numAlive / 3);
 
         let numKicked = Object.keys(this.votes).length;
+        numKicked = Math.min(numKicked, vegKickThreshold);
```

Comment:
since now more people can kick than the threshold, this prevents that chat showing 3/2 4/2 and caps it at 2/2